contracts/CourtPresaleActivate.sol

```diff
@@ -134,13 +136,34 @@ contract CourtPresaleActivate is IsContract, ApproveAndCallFallBack {
 
         // buy in presale
         _buyAndActivate(msg.sender, contributionTokenAmount, contributionToken);
+    }
+
+    /**
+    * @notice Refunds accidentally sent ETH. Only governor can do it
+    * @param _recipient Address to send funds to
+    * @param _amount Amount to be refunded
+    */
+    function refundEth(address payable _recipient, uint256 _amount) external onlyGovernor {
+        require(_amount > 0, ERROR_ZERO_AMOUNT);
+        uint256 selfBalance = address(this).balance;
+        require(selfBalance >= _amount, ERROR_NOT_ENOUGH_BALANCE);
+
+        (bool result,) = _recipient.call.value(_amount)("");
```

Comment:
This is a small thing, but with solc 0.5 do you need the `("")` or can you get away with `()`?

Just seems a bit weird but I can also see making the case for using `""` to make it more explicit.

Reply:
Yes, without it I get:
```
Error: CompileError: /home/bingen/workspace/court_presale_activate/contracts/CourtPresaleActivate.sol:140:26: TypeError: Wrong argument count for function call: 0 arguments given but expected 1. This function requires a single bytes argument. Use "" as argument to provide empty calldata.
        (bool result,) = _recipient.call.value(_amount)();
                         ^------------------------------^
```